src/Controller/LoginController.php

```diff
@@ -17,6 +21,10 @@ function __construct($connection, $passwordHasher) {
         $this->passwordHasher = $passwordHasher;
     }
 
+    /**
+     * 
+     * @return NULL
```

Comment:
Kartais gali sutikti kad kas nors aprašinėja `@return void`, bet tai nėra labai teisingas/pripažįstamas variantas. Jeigu nieko funkcija negrąžina - nedėkim ir `@return NULL`.